soundbay/conf/preprocessors/wav2vec2.yaml

```diff
@@ -0,0 +1,2 @@
+# @package _global_
+_preprocessors: []
```

Comment:
if there are no preprocessors why is this file needed and not use the `defaults`?

Reply:
wav2vec is working on the raw input (the waveform) and the defaults file has a preprocess of the data into a spectrogram.